Errors are reported twice

It seems that currently any error is reported twice:

```
  22 passing (13s)
  1 failing

  1) eslint should have no errors in lib:
     Code did not pass lint rules
/Users/tbn/Code/ember-cli/lib/utilities/reexport-template.js
  2:1  error  Use the global form of "use strict"  strict
  3:3  error  Use the global form of "use strict"  strict

✖ 2 problems (2 errors, 0 warnings)

  Error: Code did not pass lint rules
  lib/utilities/reexport-template.js
    2:1  error  Use the global form of "use strict"  strict
    3:3  error  Use the global form of "use strict"  strict

  ✖ 2 problems (2 errors, 0 warnings)

      at Context.<anonymous> (node_modules/mocha-eslint/index.js:27:13)
```

Interesting about this is that one of the outputs is using relative paths, while the other is using the absolute path of the file.

--- a/index.js
+++ b/index.js
@@ -31,7 +31,8 @@ function test(p, opts) {
     ) {
       throw new Error(
         chalk.red('Code did not pass lint rules') +
-        formatter(report.results)
+        // remove process.cwd() to convert absolute to relative paths
+        formatter(report.results).replace(process.cwd() + '/', '')
       );
     } else if (
       warn &&

--- a/tests/acceptance/acceptanceTest.js
+++ b/tests/acceptance/acceptanceTest.js
@@ -21,6 +21,15 @@ describe('Acceptance: mocha-eslint', function () {
       if (results[4].indexOf('1 failing') === -1) {
         throw new Error('Did not get a failing test');
       }
+
+      var reasonsCount = results[6].split('\n')
+          .filter(function(line) { return line.indexOf('Code did not pass lint rules') !== -1; })
+          .length;
+
+      if (reasonsCount !== 1) {
+        throw new Error('Counted ' + reasonsCount + " failure reasons");
+      }
+
       done();
     });
   });

